gateway-ha/src/test/java/io/trino/gateway/ha/HaGatewayTestUtils.java

```diff
@@ -77,17 +66,26 @@ public static void prepareMockBackend(
                 .setResponseCode(200));
     }
 
-    public static TestConfig buildGatewayConfigAndSeedDb(int routerPort, String configFile)
-            throws Exception
+    public static void seedRequiredData(String h2DbFilePath)
     {
-        File tempH2DbDir = Path.of(System.getProperty("java.io.tmpdir"), "h2db-" + RANDOM.nextInt() + System.currentTimeMillis()).toFile();
-        tempH2DbDir.deleteOnExit();
+        String jdbcUrl = "jdbc:h2:" + h2DbFilePath;
+        Jdbi jdbi = Jdbi.create(jdbcUrl, "sa", "sa");
+        try (Handle handle = jdbi.open()) {
+            handle.createUpdate(HaGatewayTestUtils.getResourceFileContent("gateway-ha-persistence-mysql.sql"))
+                    .execute();
+        }
+    }
 
+    public static File buildGatewayConfig(int routerPort, String configFile, PostgreSQLContainer postgresqlContainer)
```

Comment:
Rename `postgresqlContainer` to `postgreSqlContainer` and move to the 1st argument. It's a kind of service objects.